src/main/java/no/nav/pensjon/selvbetjeningopptjening/fullmakt/client/FullmaktClient.java

```diff
@@ -35,39 +47,37 @@ public FullmaktClient(WebClient webClient,
         this.url = requireNonNull(baseUrl, "baseUrl");
     }
 
-    public FullmaktsforholdDto harFullmaktsforhold(String fullmaktsgiverPid, String fullmektigPid) {
-        if (log.isDebugEnabled()) {
-            log.debug("Calling {} for PID {}", SERVICE, maskFnr(fullmektigPid));
-        }
+    public RepresentasjonValidity hasValidRepresentasjonsforhold(String fullmaktsgiverPid) {
 
         try {
             return webClient
                     .get()
                     .uri(url())
-                    .headers(h -> setHeaders(h, fullmaktsgiverPid, fullmektigPid))
+                    .headers(h -> setHeaders(h, fullmaktsgiverPid))
```

Comment:
Hentes `fullmektigPid` fra JWT-en isteden?

Reply:
Ja, den nye pdl-fullmakt-tjenesten som vi på sikt skal gå mot støtter kun å hente pid fra tokenet, så da tilpasset jeg vår tjeneste til også å følge samme mønster. De kunne nok ha lagt til støtte for at det kunne spesifiseres på annen måte også, men tenker at fullmakt egentlig kun gir mening i forbindelse med eksterne brukere, så da valgte jeg å følge samme mønster som dem.